frontend/src/pages/Login/Login.js

```diff
@@ -1,48 +1,49 @@
 import React, { useState } from 'react';
-import { Redirect } from 'react-router'
-import Logo from '../App/lhl_logo.png'
+import { Redirect } from 'react-router';
+import qs from 'query-string';
 import Axios from 'axios';
-
-// import Button from '../../external_components'
-import 'antd/dist/antd.css';
 import { Button } from 'antd'; // TODO CONNECT TO EXTERNAL COMPONENTS
-import './login.css'
-import qs from 'query-string';
+import 'antd/dist/antd.css';
```

Comment:
We shouldn't need these lines (5/6) here. `external_components` folder should already include this.

Line 5 should be `import { Button } from '../../../external_components' - I think

Reply:
Fixed in the latest push